src/components/Filter/PriceFilter.jsx

```diff
@@ -0,0 +1,23 @@
+export default function PriceFilter({ productState, productDispatch }) {
+  return (
+    <>
+      <input
```

Comment:
Suggestion: Add steps for better UX.

Reply:
Thank you for the suggestion, I was looking for something like this